### ISSUE TITLE
Were Pinecone integrations retracted from the codebase?

### ISSUE THREAD

**Issue by serhatgktp** [NVIDIA/NeMo-Guardrails#289]
At some point, I recall seeing Pinecone integrations were added to NeMo Guardrails. Were the commits erased from the codebase afterwards?

**Comment by drazvan** [comment]
Yes, we've restructured things at some point and decided to remove it. But it will be back very soon.
